doc/BBCode.md

```diff
@@ -1,208 +1,611 @@
 Friendica BBCode tags reference
 ========================
 
-* [Home](help)
-
-Inline
------
-
-
-<pre>[b]bold[/b]</pre> : <strong>bold</strong>
-
-<pre>[i]italic[/i]</pre> : <em>italic</em>
```

Comment:
I didn't intend on changing the existing behavior, so your suggestion is outside of the scope of this PR, but I see what you mean. This issue is that HTML is supposed to convey meaning, while CSS adds styling. However, the current BBCodes don't follow this separation, which leads to such dilemmas for which there's no good answer.

If we assume that `[b]` and `[i]` don't have any semantic meaning, then we should actually change `<strong>` with `<span class="bold">`  or `<span style"font-weight: bold">` and `<em>` with `<span class="italic">`  or `<span style"font-style: italic">`.